src/main/java/org/jitsi/videobridge/VideobridgeExpireThread.java

```diff
@@ -47,6 +48,13 @@
         = new RecurringRunnableExecutor(
             VideobridgeExpireThread.class.getSimpleName());
 
+    /**
+     * The executor used to expire individual {@link Channel}s.
+     */
+    private static final Executor CHANNEL_EXPIRE_EXECUTOR
```

Comment:
i thought the idea would be to have a multi-threaded executor which handled the various expirations, is it being channel-specific here just an intermediary step?

Reply:
One executor (with multiple threads) is used to expire all Channel-s (but not Content-s and Conference-s).

Reply:
right, but is the plan to have a single (multi-threaded) executor handling all (channel, content, conference) expirations?

Reply:
Oh, I misread your comment.

No, I was going to keep it like this, because expiring a Content and Conference seems simple and safe to do.

Reply:
I don't mind changing it, though.

Reply:
yeah i'd say while we're in here we might as well cover potential issues in all the areas and we'll be consistent with how things are expired.